source/integrate/mattermost-mission-collaboration-for-m365.rst

```diff
@@ -0,0 +1,157 @@
+            - **Teams web**: 5e3ce6c0-2b1f-4285-8d4b-75ee78787346
+            - **Teams app**: 1fec8e78-bce4-4aaf-ab1b-5451cc387264
+            - **Outlook desktop** : d3590ed6-52b3-4102-aeff-aad2292ab01c
+            - **Outlook web**     : bc59ab01-8403-45c6-8796-ac3ef710b3e3
+            
+
+    - If you want to make your application available in more Microsoft applications, keep adding client applications from `the following table <https://learn.microsoft.com/en-us/microsoftteams/platform/tabs/how-to/authentication/tab-sso-register-aad#to-configure-authorized-client-application:~:text=Select%20one%20of%20the%20following%20client%20IDs%3A>`_.
+
+Mattermost plugin setup
+~~~~~~~~~~~~~~~~~~~~~~~
+
+1. Download the `latest release of the plugin <https://github.com/mattermost/mattermost-plugin-msteams-devsecops/releases>`_.
+
+2. Go to **System Console > Plugins > Plugin Management > Upload Plugin**, and upload the plugin binary you downloaded in the previous step.
+
+3. Go to **System Console > Plugins > Plugin Management**. In the **Installed Plugins** section, scroll to **MSTeams DevSecOps**.
+
+4. Enter an **Application Version**. You can start with ``1.0.0``.
+
+5. Generate an Application ID in `version 4 UUID format <https://www.uuidgenerator.net/>`_ and enter it in the **Application ID** field.
+
+6. Enter the values you noted earlier in the appropriate fields, including **Directory (tenant) ID**, **Application (client) ID**, and **Client Secret**. 
+
+    .. image:: ../images/tenant-client-secret-sysconsole.png
+      :alt: In the Mattermost System Console, enter the Directory (tenant) ID, Application (client) ID, and Client Secret for the plugin.
+
+7. Enter an **Application Display Name** to define how your application is named in the MS Teams App Store.
+
+8. Save the changes and enable the plugin.
+
+9. Select the **Download Manifest** button to generate the MS Teams application as a ZIP file, containing the app manifest. Save this file as it will be used in the next steps.
+
+
+Install Mattermost in Microsoft Teams
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+1. Go to the `Microsoft Teams admin center <https://admin.teams.microsoft.com/dashboard>`_.
+
+2. Go to **Teams apps > Manage apps**.
+
+    .. image:: ../images/select-manage-apps.png
+      :alt: In MS Teams, go to Manage Apps to install the Mattermost Mission Collaboration for Microsoft 365 app.
+
+3. Go to **Actions > Upload new app** located in the upper-right corner of the Manage apps page.
+
+4. Select **Upload** and select the ZIP file saved previously.
+
+5. Done! Your application is now available to users.
+
+Usage
+------
+
+This plugin bypasses standard authentication methods, enabling automatic authentication when logged into Microsoft Teams. Teams authentication automatically logs users into Mattermost if the email addresses in both platforms match exactly. Regular authentication methods (LDAP, SAML, email/password, OpenID) can additionally be used for Mattermost.
```

Comment:
I think saying we "bypass" authentication creates uneasy feeling for the admins.  Can we say `This plugin supports automatic authentication when logged into Microsoft Teams.` ?